frontend/ui/data/creoptions.lua

```diff
@@ -375,6 +378,8 @@ Note that your selected font size is not affected by this setting.]]),
                   value_hold_step = 5,
                 },
                 event = "SetLineSpace",
+                verbosity = 20,
+                more_verbosity = 30,
```

Comment:
You mean the verbosity is inherent to the Event itself? (in which case, yeah, just punt it to the handler)

Or does it depend on who sent it? (in which case, extra event arg?)

Reply:
> Or does it depend on who sent it? (in which case, extra event arg?)

I'd like to avoid an extra event arg (our event are already complex/convoluted enough :)
I don't think it depends on who sent it: who sent it is known to the notification manager that the event handler calls with `Notification:notify("This stuff changed to this or that")`. So, the notification manager can do the decision, possibly depending on another level/criticity given:
`Notification:notify("This stuff changed to this or that", Notification.LEVEL_IMPORTANT)`

> You mean the verbosity is inherent to the Event itself?

I think so.
Having verbosity= only in creoptions sets some level only when the event is sent from the bottom menu. It's for now the only place where we might want to mask some, but well, I think it's related to the event (and the other UI stuff this event handler might do, so it if does some busy UI refreshes, it might say its notification has a low level so it's more easily masked).